runtime/lua/vim/lsp/util.lua

```diff
@@ -443,6 +443,7 @@ function M.apply_text_edits(text_edits, bufnr, offset_encoding)
     }
   end)()
 
+  local is_windows = vim.loop.os_uname().sysname == 'Windows'
```

Comment:
Equality check might not work on some windows systems as `sysname` is not exactly `Windows`

```
{
  machine = "x86_64",
  release = "10.0.22000",
  sysname = "Windows_NT",
  version = "Windows 10 Enterprise"
}
```

Reply:
All right. I haven't used windows for a long time :)